apps/backend/src/users/users.service.ts

```diff
@@ -140,6 +142,16 @@ export class UsersService {
         'Cannot destroy only administrator account, please promote another user to administrator first'
       );
     }
+    // Clean up groups owned by user
```

Comment:
can you simplify the code ni lines 121-136?  there's no point doing a try-catch to catch an exception that you yourself are throwing.

i would probably do something like:

`if(abac.cannot && !(await compare)) { throw new ForbiddenException('your password was wrong') }`

Reply:
Done